packages/falcon-shop-data/src/Customer/CustomerWithAddressesQuery.tsx

```diff
@@ -28,7 +28,27 @@ export const GET_CUSTOMER_WITH_ADDRESSES = gql`
   }
 `;
 
-export class CustomerWithAddressesQuery extends Query<Customer> {
+export type CustomerWithAddressesResponse = {
+  customer: Pick<Customer, 'id' | 'firstname' | 'lastname' | 'email'> & {
+    addresses: Pick<
+      Address,
+      | 'company'
```

Comment:
Just asking, this `|` should not be after field name?
```suggestion
      'company' |
```

found this: [ts-multipick](https://github.com/staltz/ts-multipick), I am not saying that we should use it but I whant to share it here

Reply:
This actually is autoformatted by our current config, so I'd leave it as it is.